Gemfile.lock

```diff
@@ -616,69 +619,68 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
```

Comment:
### [sexp_processor](https://github.com/seattlerb/sexp_processor)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.12.1 → 4.17.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/sexp_processor/4.12.1/4.17.1)_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/seattlerb/sexp_processor/compare/v4.12.1...v4.17.1).

These are the first 10 commits:
- (1cf9bfa) [+ Added Sexp.q (query) and deprecated Sexp.s to distinguish …](https://github.com/seattlerb/sexp_processor/commit/1cf9bfa062769420c60d94f9192d56835af1d508)
- (1c679b5) [- Fixed bug w/ ruby's Array#eql? and #hash not looking at iv…](https://github.com/seattlerb/sexp_processor/commit/1c679b548d388a8cbee95b2cfe8fe4808b76baf2)
- (9d637ad) [+ Split out all pattern-oriented code to sexp_matcher.rb.](https://github.com/seattlerb/sexp_processor/commit/9d637add9a04b45178de68897bfa379b338d5ba0)
- (b9b099e) [+ Extended Sexp::Matcher::Parser to cover more method names.](https://github.com/seattlerb/sexp_processor/commit/b9b099ed9884d366b622277a2ea33c5d2e4d4d73)
- (1e78662) [prepped for release](https://github.com/seattlerb/sexp_processor/commit/1e78662595d99786fa0eeb1d80887bf7c4aa1f18)
- (52db357) [Removed old maglev skips in test](https://github.com/seattlerb/sexp_processor/commit/52db35748c18e1a3ced84950cd5e54be07acc7b7)
- (5987c8d) [+ Added Klass matcher to match on types.](https://github.com/seattlerb/sexp_processor/commit/5987c8dc0452b7444c6872e5e2f410209d2dc3a2)
- (d43ca07) [+ Added '-' as an alias for the 'not?' pattern matching comm…](https://github.com/seattlerb/sexp_processor/commit/d43ca0703e246c1a51ba44eddecc6bced7d5b655)
- (17621e8) [+ Added any matcher to pattern parser.](https://github.com/seattlerb/sexp_processor/commit/17621e82596c886bba70054d18cc05fdb8c63962)
- (4660187) [prepped for release](https://github.com/seattlerb/sexp_processor/commit/4660187826856e1370f0936b6f15948254e54f12)


</details>